Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebHitTestResult.cpp

```diff
@@ -72,15 +72,7 @@ struct _WebKitWebHitTestResultPrivate {
 #endif // ENABLE(2022_GLIB_API)
 };
 
-#if ENABLE(2022_GLIB_API)
-struct _WebKitWebHitTestResultClass {
-    GObjectClass parent;
-};
-
 WEBKIT_DEFINE_FINAL_TYPE(WebKitWebHitTestResult, webkit_web_hit_test_result, G_TYPE_OBJECT)
-#else
-WEBKIT_DEFINE_TYPE(WebKitWebHitTestResult, webkit_web_hit_test_result, WEBKIT_TYPE_HIT_TEST_RESULT)
-#endif
```

Comment:
Oops, careful here! You broke WebKitWebHitTestResult in the pre-2022 API.

Reply:
Good catch, thanks! I was a bit trigger-happy removing things around 😉